src/data/staking-products.json

```
@@ -356,6 +356,39 @@
     }
   ],
   "saas": [
+    {
+      "name": "ChainLabo",
+      "imageName": "ChainLabo",
+      "hue": 248,
+      "launchDate": "2023-04-01",
+      "url": "https://www.chainlabo.com/",
+      "audits": [],
+      "minEth": 32,
+      "additionalStake": null,
+      "additionalStakeUnit": null,
+      "monthlyFee": 200,
+      "monthlyFeeUnit": "CHF",
+      "isFoss": false,
+      "hasBugBounty": false,
+      "isTrustless": true,
+      "isPermissionless": false,
+      "pctMajorityExecutionClient": 100,
+      "pctMajorityConsensusClient": 100,
```

@natpicone You mentioned use of Nimbus, correct?
```suggestion
      "pctMajorityConsensusClient": 0,
```

---

Thanks, I missed this